elements/bulbs-clickventure/clickventure.js

```diff
@@ -228,4 +236,12 @@ export default class Clickventure {
       this.adsManager.reloadAds(this.sideAd);
     }
   }
+
+  sendPageView () {
+    this.analyticsManager.trackPageView(
+      this.href,
```

Comment:
Also, re other comments, `this.href` argument should be the boolean `false` here